### PR TITLE
Replace space with underscore in member file names

### PR DESCRIPTION
**Pull request by leeping**
This makes it easier to work with the source tree in the shell / command line utilities such as find.

### CODE REVIEW

File: docs/sitemap.xml [hunk 1]
```diff
@@ -46,13 +46,13 @@
   </url>
   
   <url>
-    <loc>//openforcefield.org/tags/news/</loc>
+    <loc>//openforcefield.org/categories/news/</loc>
```
[the remark this answers is not in the document]
Comment (leeping) [reply docs/sitemap.xml]: I'm not sure.. that change came from building the site.

**Comment by leeping** [comment]
(1) Thanks.
(2) I did rebuild, this was the only change in `docs/`.
(3) Yes, I verified the result with a local preview.

File: docs/sitemap.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (leeping) [reply docs/sitemap.xml]: In the local preview, `tags/news` and `categories/news` points to two very similar but not identical pages. @jchodera do you know what's going on?

[the remark this answers is not in the document]
Comment (jchodera) [reply docs/sitemap.xml]: Not sure, but each news item has both tags and categories in the header YAML that we can list. They're just arbitrary keywords that automatically get collected.

**Pull request merged**